packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -346,8 +346,11 @@ async function createWatchers(dirs: string[], cb: (files: string[]) => void) {
           if (event.type === 'delete') return
 
           // Ignore directory changes. We only care about file changes
-          let stats = await fs.lstat(event.path)
-          if (stats.isDirectory()) {
+          let stats: Stats | null = null
+          try {
+            stats = await fs.lstat(event.path)
+          } catch {}
+          if (stats === null || stats.isDirectory()) {
```

Comment:
```suggestion
          if (!stats || stats.isDirectory()) {
```

Reply:
I wonder if we could change this to something like `if (!stats?.isFile()) {`

Not sure if that would break things tho

Reply:
Seems correct yep

Reply:
hm maybe actually not, we probably want to follow symlinks?